src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -137,16 +138,15 @@ private static void usage() {
   public static void main(final String[] args) {
     ErrorConsole.getConsole();
 
-    try (PerfTimer timer = Perf.startTimer("Show main window")) {
-      // do after we handle command line args
-      checkForMemoryXMX();
-      SwingUtilities.invokeLater(() -> setupLookAndFeel());
-      showMainFrame();
-    }
-    (new Thread(() -> setupLogging())).start();
-
+    PerfTimer timer = PerfTimer.startTimer("Show main window");
+    // do after we handle command line args
+    checkForMemoryXMX();
+    SwingUtilities.invokeLater(() -> setupLookAndFeel());
+    showMainFrame();
+    new Thread(() -> setupLogging()).start();
+    timer.stop();
```

Comment:
The variable is needed so java has something to call close on. Ideally we would not even need that. The try/with block is less error prone, there is no way to forget to call the 'close'. 

But, also, there is another functional change, the timer.close() is no longer called in a finally. If the timed operation were to crash, the timer would never be stopped.